brave/browser/component_updater/brave_component_updater_configurator.cc

```diff
@@ -96,10 +98,15 @@ int BraveConfigurator::UpdateDelay() const {
 
 std::vector<GURL> BraveConfigurator::UpdateUrl() const {
   if (use_brave_server_) {
-    // For localhost of vault-updater
-    // return std::vector<GURL> {GURL("http://localhost:8192/extensions")};
-    return std::vector<GURL>
-        {GURL("https://laptop-updates.brave.com/extensions")};
+    std::unique_ptr<base::Environment> env(base::Environment::Create());
+    // Set this variable if you wanna use a local extension server
+    if (env->HasVar("LOCAL_EXTENSION_SERVER")) {
```

Comment:
actually an env var could create a security problem. I think it's probably better to pass a command line option for this